frontend/src/packages/auth/auth-api.ts

```diff
@@ -35,6 +36,19 @@ class AuthApi extends BaseHttpApi {
 
     return await response.json<UserSignUpResponseDto>();
   }
+
+  public async getUser(): Promise<UserGetAllItemResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(AuthApiPath.USER, {}),
+      {
+        method: 'GET',
+        contentType: ContentType.JSON,
```

Comment:
We don't send body
```suggestion
```

Reply:
hey - but contentType is a mandatory field, so I can't delete it from request - should I update the HTTPApiOptions type or leave it as it is?

```
type HTTPApiOptions = Omit<HTTPOptions, 'headers' | 'payload'> & {
  hasAuth: boolean;
  contentType: ValueOf<typeof ContentType>;
  payload?: HTTPOptions['payload'];
};
```

Reply:
make this field optional please 🙏